src/pages/PassportRedemption/SignIn/uploadScreen.tsx

```diff
@@ -0,0 +1,302 @@
+    event.preventDefault();
+    if (event.target.files && event.target.files[0]) {
+      setReceiptFilePath(URL.createObjectURL(event.target.files[0]));
+      setReceipt(event.target.files[0]);
+    }
+  };
+
+  const submitReceipt = async () => {
+    if (receipt === null) return false;
+
+    try {
+      const ext = receipt.type.split('/')[1];
+
+      let filename = `${id}-${new Date()
+        .toISOString()
+        .replace(/(:|\.)/g, '-')}.${ext}-${participatingSeller}`;
+
+      // upload receipt to gc
+      const signedUrl = unescape(
+        (await getUploadUrl(filename, receipt.type)).data.url
+      );
+      await sendImage(signedUrl, filename, receipt);
+
+      // upload info + receipt to db
+      let data = await uploadCrawlReceipts(
+        participatingSellerId,
+        id,
+        Number(billTotal),
+        filename
+      );
+
+      if (data) push(`/lny-passport/${id}/tickets`);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  return (
+    <>
+      <Column>
+        <SubTitle>{t('passport.labels.showReceipt')}</SubTitle>
+        <Column />
+        <Label htmlFor="crawl-merchant-name">
+          {t('passport.labels.crawlMerchant')}
+        </Label>
+        <CustomSymbolContainer className="allParticipatingSellers">
+          <InputField
+            name="crawl-merchant-name"
+            type="text"
+            onChange={(e) => {
+              e.preventDefault();
+              setParticipatingSeller(e.target.value);
+              setSelectedSeller(false);
+            }}
+            value={participatingSeller}
+            placeholder="Search Chinatown Businesses"
+          />
+        </CustomSymbolContainer>
+
+        {!!participatingSeller && !selectedSeller ? (
+          !!sellersSearchBar.length ? (
+            <SearchContainer>
+              {sellersSearchBar.map((merchant) => (
+                <SearchResult
+                  key={merchant.name + 'lny-crawl'}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    setParticipatingSeller(merchant.name);
+                    setSelectedSeller(true);
+                    setParticipatingSellerId(merchant.id);
+                  }}
+                >
+                  {merchant.name}
+                </SearchResult>
+              ))}
+            </SearchContainer>
+          ) : (
+            <SearchContainer className="addPadding">
+              {t('passport.errors.validParticipatingSeller')}
+            </SearchContainer>
+          )
+        ) : null}
+
+        <Label htmlFor="bill-total">{t('passport.labels.billTotal')}</Label>
+        <CustomSymbolContainer className="billTotal">
+          <InputField
+            name="bill-total"
+            className="indent"
+            type="number"
+            onChange={(e) => {
+              e.preventDefault();
+              setBillTotal(e.target.value);
+            }}
+            value={billTotal}
+            min="10"
```

Comment:
im not sure when you plan to convert the amount to cents, but just making sure you know to use cents instead of dollars

Reply:
good call out, def did not know to make it cents hahahha

Reply:
im going to put the dollar --> cents conversion when we send it to the b/e. not going to edit it here since this is also tied to the value that is displayed to the user